### PR TITLE
CSS: more pseudo-element tests

### PR DESCRIPTION
**Pull request by annevk**
Test ::-webkit-autofill in @supports and add more getComputedStyle() coverage.

### CODE REVIEW

File: css/css-conditional/at-supports-selector-webkit-autofill.optional.html
Comment (nt1m) [inline css/css-conditional/at-supports-selector-webkit-autofill.optional.html:1]: I know the whole directory is tab-indented (and there is a linter exception for this dir) but we should probably avoid it.

File: css/cssom/getComputedStyle-pseudo.html [hunk 1]
```diff
@@ -105,24 +105,41 @@
   <ul><li id="pseudo-invalid">Item</li></ul>
 </div>
 <script>
-test(function() {
-  var div = document.getElementById('test');
-  [":before", ":after"].forEach(function(pseudo) {
+test(() => {
+  const div = document.getElementById('test');
+  ["before", "after"].forEach(pseudo => {
+    assert_equals(getComputedStyle(div, pseudo).width, "100px");
```
Comment (nt1m) [inline css/cssom/getComputedStyle-pseudo.html:111]:
```suggestion
    assert_not_equals(getComputedStyle(div, pseudo).width, "100px");
```

If this is invalid (per subtest name), shouldn't this be not_equals ? or equals to empty string?

**Review by nt1m — APPROVED** [review]

File: css/cssom/getComputedStyle-pseudo.html (hunk 1 above)
Comment (annevk) [reply css/cssom/getComputedStyle-pseudo.html:111]: getComputedStyle has this weird thing that when its pseudo-element does not start with a colon it's supposed to be ignored.

Comment (nt1m) [reply css/cssom/getComputedStyle-pseudo.html:111]: Ah yeah I see this in the algorithm, perhaps you could make the subtest message more clear then, or add a comment to clearly distinguish the invalid case and the no-colon case

File: css/css-conditional/at-supports-selector-webkit-autofill.optional.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<title>CSS Test (Conditional Rules): In @supports, known -webkit- pseudo elements can be parsed successfully</title>
+<link rel="help" href="http://www.w3.org/TR/css3-conditional/#at-supports">
```
Comment (nt1m) [inline css/css-conditional/at-supports-selector-webkit-autofill.optional.html:3]: Can you link to drafts.csswg.org ?

File: css/css-conditional/at-supports-selector-webkit-autofill.optional.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<title>CSS Test (Conditional Rules): In @supports, known -webkit- pseudo elements can be parsed successfully</title>
+<link rel="match" href="at-supports-001-ref.html">
+<style>
+  div {
+    background:red;
+    height:100px;
+    width:100px;
+  }
+  @supports selector(input::-webkit-autofill) {
```
Comment (nt1m) [inline css/css-conditional/at-supports-selector-webkit-autofill.optional.html:10]: Wait, I just noticed this isn't actually a known `-webkit-` pseudo-element. `:-webkit-autofill` is a pseudo-class, not a pseudo-element.

If you want to test an unknown `::-webkit-` pseudo-element, it should be `::-webkit-unknown-pseudo-element` or something.

**Review by nt1m — CHANGES_REQUESTED** [review]
See comment above, also the linter failure should be fixed.

File: css/css-conditional/at-supports-selector-webkit-autofill.optional.html (hunk 3 above)
Comment (annevk) [reply css/css-conditional/at-supports-selector-webkit-autofill.optional.html:11]: I want to test a known one. Hmm.

Comment (annevk) [reply css/css-conditional/at-supports-selector-webkit-autofill.optional.html:11]: I guess all pseudo-elements are proprietary at the moment so this only matters for browsers supporting a -webkit-prefixed pseudo-element. I guess I could pick one of the ones Chromium and WebKit both have.

**Review by nt1m — APPROVED** [review]

File: css/css-conditional/at-supports-selector-webkit-slider-thumb.tentative.html
Comment (nt1m) [inline css/css-conditional/at-supports-selector-webkit-slider-thumb.tentative.html:1]: I think optional might be better than tentative, unless there's a plan to standardize this as a legacy alias

**Pull request merged**